w04-associations-and-auth/d3-homework/README.md

```diff
@@ -0,0 +1,122 @@
+# Project 1 Planning
+
+## [Requirements](https://github.com/sf-wdi-22-23/modules/tree/master/w04-associations-and-auth/project-1)
```

Comment:
@mtvillwock I got paused in my review.  Yes, and if you want I can just make the tweaks I'm finding and tag you if there are any blocking questions.  I'll get back to it here in about 20min